src/components/modal/index.tsx

```diff
@@ -73,16 +64,14 @@ class Modal extends Component<ModalProps> {
   }
 
   render() {
-    const {blurView, enableModalBlur, visible, ...others} = this.props;
-    const defaultContainer = enableModalBlur && Constants.isIOS ? BlurView : View;
-    const Container: any = blurView ? blurView : defaultContainer;
+    const {visible, ...others} = this.props;
 
     return (
       <RNModal visible={Boolean(visible)} {...others}>
-        <Container style={{flex: 1}} blurType="light">
+        <View style={{flex: 1}}>
```

Comment:
Do we really need this View? Is the touchableOverlay can't be the container view?

Reply:
I think it can simply be removed, tested on the places I could think of and it looked ok.